src/inner-slider.js

```diff
@@ -751,6 +751,7 @@ export class InnerSlider extends React.Component {
 
     if (this.props.unslick) {
       listProps = { className: "slick-list" };
+      trackProps.trackStyle = null;
```

Comment:
We should change this line: https://github.com/ant-design/react-slick/blob/3343a0d00fb5bf16edee1a78608e6702be17aae3/src/utils/innerSliderUtils.js#L594

to 

```js
  const trackChildren = spec.unslick ? spec.slideCount : spec.slideCount + 2 * spec.slidesToShow;
```

Reply:
ok